components/x11/glw/Makefile

```diff
@@ -35,10 +36,11 @@ PATCH_LEVEL=0
 
 LDFLAGS= -L$(X11_SERVERLIBS_DIR) -R$(X11_SERVERLIBS_DIR)
 
-CONFIGURE_OPTIONS += --enable-motif
+#CONFIGURE_OPTIONS += --enable-motif
+REQUIRED_PACKAGES += developer/build/autoconf/xorg-macros
```

Comment:
When you comment out something you should leave a comment why you did it. Otherwise others have to guess.
Especially when someone updates the version such relics aren't helpful. Thus, either through it away or leave it like that with a comment. In any way you should explain why you did it in the commit. Otherwise this looks like an arbitrary change.
Furthermore, when you change the configuration for a package make sure it still builds all files that are in the manifest. You should run gmake sample-manifest for packages you change the configuration for. And also if a package hasn't been updated for a long time. Our environment changes (we add, remove or modify other packages) and the build system might detect more or less packages than earlier.
You will lessen the burden of doing this for me if you write a comment in the PR what you already did. If there is no comment I will need to do that and this is a lot of work for a single person. Alas it is quite seldom that somebody else tests and comments PR's.